src/HTML5/Parser/DOMTreeBuilder.php

```diff
@@ -474,8 +474,14 @@ public function endTag($name)
     {
         $lname = $this->normalizeTagName($name);
 
-        // Ignore closing tags for unary elements.
-        if (Elements::isA($name, Elements::VOID_TAG)) {
+        // Special case within 12.2.6.4.7: An end tag whose tag name is "br" should be treated as an opening tag
+        if ($name === 'br') {
+            $this->parseError('Closing tag encountered for void element br.');
```

Comment:
I'm not sure that triggering a parse error is needed. Will the returned DOM still be as expected?

Reply:
I was just going by the wording of the spec, which has [a specific definition of "parse error"](https://html.spec.whatwg.org/multipage/parsing.html#parse-errors) in terms of declaring a document non-conformant, even where the result is well-defined. 

The DOM should look exactly as though this was a normal void `<br>` tag with no attributes, so I'm happy to remove this line if you want to limit `parseError` to more serious problems.

Reply:
We already have parse errors for stuff like duplicate IDs, so ...

Reply:
:+1: